test/hdnode.js

```diff
@@ -124,12 +123,12 @@ describe('HDNode', function () {
     })
 
     describe('getAddress', function () {
-      it('wraps keyPair.getAddress', setupTest(function () {
-        this.mock(keyPair).expects('getAddress')
-          .once().withArgs().returns('foobar')
-
-        assert.strictEqual(hd.getAddress(), 'foobar')
-      }))
+      // it('wraps keyPair.getAddress', setupTest(function () {
+      //   this.mock(keyPair).expects('getAddress')
+      //     .once().withArgs().returns('foobar')
+      //
+      //   assert.strictEqual(hd.getAddress(), 'foobar')
+      // }))
```

Comment:
Should discuss whether keys have an associated address type. Maybe following keytoscript, we should just assume _some_ function is responsible for converting keys to their scripts, and then addresses, rather than having getAddress and trying to manage defaults.